Tasks/Xcode/task.json

```diff
@@ -11,8 +11,8 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 4,
-        "Minor": 126,
-        "Patch": 7
+        "Minor": 127,
```

Comment:
You will have to port to releases/127 since the release branch was cut off. Otherwise change minor version to 128.

Reply:
I'll port. I considered 128, but it won't propagate until mid-January by default.